sys-apps/smartmontools/smartmontools-6.4-r1.ebuild

```diff
@@ -72,11 +74,11 @@ src_install() {
 		newconfd "${FILESDIR}"/smartd.confd smartd
 
 		# Move drivedb.h file out of PM's sight (bug #575292)
-		mv "${ED}"${db_path}/drivedb.h "${T}" || die
-		keepdir ${db_path}
+		mv "${ED}${db_path}/drivedb.h" "${T}" || die
+		keepdir "${db_path}"
 
 		exeinto /etc/cron.monthly
-		doexe "${FILESDIR}"/${PN}-update-drivedb
+		doexe "${FILESDIR}/${UPDATE_DRIVEDB}"
```

Comment:
Why are you trying to install this cron script with different name but did not either

- change the file's name in FILESDIR

or

- use `newexe` to rename the file on installation
Please fix!